### ISSUE TITLE
Questions about the execution time in the experimental part

### ISSUE THREAD

**Issue by Ubee121** [SYSU-STAR/H2-Mapping#33]
Thank you for your outstanding work! I have a small question about the experimental part. When I run the Replica dataset, the resolution is 1200x680, while for ScanNet, it's 640x480. Why is the execution time per frame higher for ScanNet than for Replica?
![Uploading image.png…]()

**Comment by JIANG-CX** [comment]
Thank you for your question. Since we limit the number of samples and do not sample all the pixels during training, differences in image resolution do not affect the training time. In the case of scannet0000, there are more input images, resulting in a larger keyframe set and increased time for the coverage-maximizing keyframe strategy. Additionally, different scenes have varying geometric structures, which affect octree reconstruction and, consequently, the time required for ray casting.

**Comment by Ubee121** [comment]
Thank you for your response! I have some technical questions to consult you. Have you ever conducted tests using Jetson Nano? I encountered an error with tinycudann when I tried to deploy on Nano, specifically during the cmake --build build --config RelWithDebInfo -j command. Some people suggested that the error might be due to an outdated version of CUDA, but it seems that Nano supports up to JetPack 4.Thanks again !
